### ISSUE TITLE
Analytics Implementation for the VA.gov Profile: Prompt to ask people if they want to update mailing address when they change home address

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#40107]
# Google Analytics Implementation or QA Request

### Required
**Is this request for **Google Analytics implementation** or **QA**?**

- [X] Google Analytics implementation
- [ ] QA

**Who is this request for?**

This request is for the profile half of the authenticated experience team. We just implemented functionality that will ask people if they also want to update their mailing address when they update their home address in the VA.gov profile. The mailing address is where all VA correspondence and prescriptions go, but some people don't realize that their home and mailing addresses are treated differently within the VA. This is why we added this prompt. We'd like to add tracking to detect how many people interact with this prompt.

Frontend work was in #33774
    
**Provide the link(s) for Google Analytics implementation or QA**
  
This feature lives at https://staging.va.gov/profile/contact-information

- Log in to VA.gov staging and go to the profile
- Go to the Contact information tab
- Update the home address to be something different that the mailing address
- You should see this prompt
    
**When is the launch date?**

- ASAP. We're waiting on GA implementation and to finish up a few accessibility tasks.

**What interactions need to be tracked?** _NOTE: [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) are automatically tracked_
    
- How many people click "Yes" in the modal below
- How many people click "No" in the modal below

![image](https://user-images.githubusercontent.com/34068740/163875192-13128cf2-6c7a-4eb6-a391-6e9f8464a967.png)

### Optional

All LOA3 staging users have access to the profile and have access to this feature, so you can use whoever you want. 

> Please leave the following blank

## Acceptance criteria
- [ ] All appropriate issue tagging is completed
- [ ] QA has been completed
- [ ] Events have been added to `Product` and `Benefit Hub` Content Groups

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Analytics Request Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html) on Platform website for more information about the Collaboration Cycle.

**Comment by Samara-Strauss** [comment]
cc/ @SKasimow @adamwhitlock1 This is an analytics request to add tracking to the address change messaging work. @adamwhitlock1 I know you'll be heads-down on the personal info work this next sprint, but wanted to get this to the analytics folks so it's ready for you once you're able to focus on the address work again.

**Comment by Samara-Strauss** [comment]
@SKasimow this is a ticket for the analytics team, so no need to add it to our sprints/milestones. Once this team is ready for help from @adamwhitlock1 to implement the tags into the frontend code, we can create a separate ticket for that and slot it into one of our sprints accordingly.

**Comment by Samara-Strauss** [comment]
@jennaouellet @jonathan-epstein13 could we get an update on this ticket? No rush, but working on planning and trying to figure out when we might be able to launch this feature. Let me know when you think you might have time to get to this. Thanks!

**Comment by SKasimow** [comment]
Hi @Samara-Strauss - do we also want to add tracking for the "x" (close) button?

**Comment by Samara-Strauss** [comment]
@SKasimow hmmm, good question. I guess that the X qualifies as a "no" response, so we could track the X and the "No" button the same way.

**Comment by jonathan-epstein13** [comment]
The event isn't automatically tracked and will need to reconfigure. Should be completed in the next day or two. Keep you posted @SKasimow. I appreciate you reaching out!

**Comment by SKasimow** [comment]
@jonathan-epstein13 thank you so much!

**Comment by jonathan-epstein13** [comment]
@SKasimow 

The modal here is not using the component so the existing modal interactions are not firing. They can either switch to the component or pass in the event manually using this taxonomy. 

'event: 'int-modal-click', 
'modal-status': this.props.status, 
'modal-title': this.props.title, 
'modal-primaryButton-text': this.props.primaryButton.text, 
'modal-secondaryButton-text': this.props.secondaryButton.text

This is how the component automatically grabs data, but since they aren’t using it - they’ll need to send the values in. The “event: int-modal-click” parameter can remain as it is, then the others should be included with the values the developer inputs.

**Comment by Samara-Strauss** [comment]
@SKasimow can you ticket this work for Adam? Also, definitely point him to @jonathan-epstein13 if he has questions since it sounds like this use case is unique.

**Comment by SKasimow** [comment]
New FE ticket #41066 opened for this bro. Closing this ticket.
